documentation/modules/deploying/proc-deploy-setup-external-clients.adoc

```diff
@@ -0,0 +1,252 @@
+// Module included in the following assemblies:
+//
+// deploying/assembly_deploy-verify.adoc
+// getting-started.adoc
+
+[id='setup-external-clients-{context}']
+= Setting up access for clients outside of Kubernetes
+
+This procedure shows how to configure client access to a Kafka cluster from outside Kubernetes.
+
+Using the address of the Kafka cluster, you can provide external access to a client on a different Kubernetes namespace or outside Kubernetes entirely.
+
+You configure an external Kafka listener to provide the access.
+
+The following external listener types are supported:
+
+* `route` to use OpenShift `Route` and the default HAProxy router
+* `loadbalancer` to use loadbalancer services
+* `nodeport` to use ports on Kubernetes nodes
+* `ingress` to use Kubernetes _Ingress_ and the {NginxIngressController}
+
+The type chosen depends on your requirements, and your environment and infrastructure.
+For example, `nodeport` is the least secure, so you might not wish to use it in a production environment.
+
+In this procedure:
+
+. An external listener is configured for the Kafka cluster, with TLS encryption and authentication, and Kafka _simple authorization_ is enabled.
+. A `KafkaUser` is created for the client, with TLS authentication and Access Control Lists (ACLs) defined for _simple authorization_.
+
+You can configure your listener to use TLS or SCRAM-SHA authentication,
+both of which can be used with TLS encryption.
+If you are using an authorization server, you can use token-based link:{BookURLUsing}#assembly-oauth-authentication_str[{oauth} authentication] and link:{BookURLUsing}#assembly-oauth-authorization_str[{oauth} authorization].
+
+When you configure the `KafkaUser` authentication and authorization mechanisms, ensure they match the equivalent Kafka configuration:
+
+* `KafkaUser.spec.authentication` matches `Kafka.spec.kafka.listeners.*.authentication`
+* `KafkaUser.spec.authorization` matches `Kafka.spec.kafka.authorization`
+
+NOTE: Authentication between Kafka users and Kafka brokers depends on the authentication settings for each.
+For example, it is not possible to authenticate a user with TLS if it is not also enabled in the Kafka configuration.
+
+Strimzi operators automate the configuration process:
+
+* The Cluster Operator creates the listeners and sets up the cluster and client certificate authority (CA) certificates to enable authentication within the Kafka cluster.
+* The User Operator creates the user representing the client and sets up the user CA certificates for secure access to the Kafka cluster.
```

Comment:
it doesn't create "CA" certificates but the private key and certificate couple for secure access so for TLS client authentication.

Reply:
I've changed to: _and the security credentials used for client authentication, based on the chosen authentication type._
We go into the details in the steps